.github/workflows/test.yml

```diff
@@ -6,6 +6,7 @@ on:
 jobs:
   build_and_test:
     name: Build and Test
+    environment: Development
```

Comment:
We have 3 separate keys for Growthbook, one for development, staging and production. We will be using the development key for test links